forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -129,6 +135,7 @@ internal class BTPinCollector(
         if (token.contains(CollectorConstants.TOKEN_DELIMITER)) {
             return token.split(CollectorConstants.TOKEN_DELIMITER)[1]
         }
+        logger.e("BT Token wasn't found on card")
```

Comment:
include `merchantAccount`,  `paymentMethod.ref` and `token` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit token